LayoutTests/accessibility/mac/html-section-elements.html

```diff
@@ -44,8 +49,10 @@
           shouldBe("body.childAtIndex(6).childAtIndex(0).subrole", "'AXSubrole: '");
 
           // When a header and footer are inside a section, they should not be landmarks.
-          // Instead, expect only the static text inside to be exposed.
-          shouldBe("body.childAtIndex(7).role", "'AXRole: AXStaticText'");
+          shouldBe("body.childAtIndex(7).childAtIndex(0).subrole", "'AXSubrole: '");
```

Comment:
This is verifying the subrole of the static text within `section > header`, right? If so, that's probably not what we want to test (should be `childAtIndex(7).childAtIndex(0)`).

Reply:
Do you want to check the value of childAtIndex(7).childAtIndex(0)? It returns a type of object.  I just wanted to check the subrole of section > header like checking article > header. Anyway, we need to check the role and subrole of both as you mentioned below.

Reply:
Sorry, my last comment was wrong (specifically the "should be" part). But the other part is accurate — `childAtIndex(7).childAtIndex(0)` is not the `section > header`, it's the static text AX object within `section > header`. If your intention is to verify the subrole of `section > header`, I believe `childAtIndex(7)` is the right one. Please add an `id` to this one and verify the `domIdentifer` to help future us understand this more easily.

Reply:
Done.